recipes/lapack/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+# CMAKE_INSTALL_LIBDIR="lib" suppresses CentOS default of lib64 (conda expects lib)
+
+cmake \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DCMAKE_INSTALL_LIBDIR="lib" \
+  -DBUILD_TESTING=OFF \
+  -DBUILD_SHARED_LIBS=ON \
+  -DLAPACKE=ON \
+  ..
+
+make -j2
```

Comment:
We should probably use 1 on Mac. So, could you please add some sort of `if` statement here? For what it is worth, you could used `CPU_COUNT` locally so a check for CI before doing this stuff would be good too.

We will have to figure out a way long term so that people don't need to rediscover this every time.